docs/files/debian-postinstall.sh

```diff
@@ -1,11 +1,43 @@
+#!/bin/sh
+
 # SPDX-FileCopyrightText: 2021 The NGI Pointer Secure-Scuttlebutt Team of 2020/2021
 #
 # SPDX-License-Identifier: CC0-1.0
 
+set -e
+
 # create a user to run the server as
 adduser --system --home /var/lib/go-ssb-room go-ssb-room
 chown go-ssb-room /var/lib/go-ssb-room
 
+if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
```

Comment:
i stole these systemd script things from [peachcloud/peach-package-builder](https://github.com/peachcloud/peach-package-builder) who stole them from [mmstick/cargo-deb](https://github.com/mmstick/cargo-deb) :package:

Reply:
however, they do change things such that the service starts automatically when installed (and is enabled to start on boot), as this is the usual behavior on Debian. if we follow this behavior, we should update the postinstall message to tell you to restart the service rather than reload daemon after making changes.

would there be a problem with the service first running with the wrong config, then changing it and restarting the service?

as a side note, we shouldn't be recommending updating services by writing to the systemd file directly, there's a whole built-in edit workflow with conf overrides and such that we should be doing.

Reply:
These changes seem legit! Thanks for adding.

> would there be a problem with the service first running with the wrong config, then changing it and restarting the service?

According to:

https://github.com/ssbc/go-ssb-room/blob/641069a8f176c5d3492b37a8c5ca6b46d1acd3fd/docs/files/debian-postinstall.sh#L26

The startup will fail? Maybe we can avoid automatically starting up & recommend the commands? Unsure what Debian does when installed packages need to be configured first?

> as a side note, we shouldn't be recommending updating services by writing to the systemd file directly, there's a whole built-in edit workflow with conf overrides and such that we should be doing: https://askubuntu.com/questions/659267/how-do-i-override-or-configure-systemd-services.

One for the docs I guess, can do here or let's open an issue to catch that another time?